modules/core/src/matrix.cpp

```diff
@@ -357,13 +357,13 @@ void Mat::create(int d, const int* _sizes, int _type)
             a = a0;
         CV_TRY
         {
-            u = a->allocate(dims, size, _type, 0, step.p, 0, USAGE_DEFAULT);
+            u = a->allocate(dims, size, _type, 0, step.p, static_cast<AccessFlag>(0), USAGE_DEFAULT);
```

Comment:
Here

Reply:
@hrnr What do you think about these? Are they correct with a zero flag?

Reply:
I think the flags are ignored here, they are used only in UMat allocator. `StdMatAllocator` allocates standard memory through `fastMalloc`, so the semantic here is `ACCESS_RW`. Whatever passed here is ignored, so I think somebody put `0` to signal it will be ignored. You can put `ACCESS_RW /* ignored */` or similar if you want.

Reply:
Great! Thank you